syntax/puppet.vim

```diff
@@ -122,7 +123,7 @@ if version >= 508 || !exists("did_puppet_syn_inits")
   endif
 
   HiLink puppetVariable             Identifier
-  HiLink puppetType                 Identifier
+  HiLink puppetType                 Define
```

Comment:
If you could make the change to your branch and push it up, that'd be great as it'll keep the history cleaner.